### ISSUE TITLE
Not works in WordPress 5.0

### ISSUE THREAD

**Issue by webfalcon** [WordPress/gutenberg-examples#48]
I installed the installable version and it just brakes the dashboard.

**Comment by ajitbohra** [comment]
yes, can confirm it's not working with `WP 5.0`

`Fatal error: Uncaught Error: Call to undefined function gutenberg_get_jed_locale_data() `

cc: @swissspidy

**Comment by swissspidy** [comment]
Hi all 👋 

The function is now called `wp_get_jed_locale_data` in core.

I would add a `function_exists()` check for each function before trying to call them.

With the migration of most of the PHP functions from Gutenberg to core, this might not be the only case where this is needed.

It's also worth noting that `wp_get_jed_locale_data()` isn't the best way to add i18n support for your JavaScript files. I recommend following https://core.trac.wordpress.org/ticket/45103 and the make/core blog for any progress in this field.

**Comment by ajitbohra** [comment]
Thanks for the info @swissspidy will keep watch on both the resources mentioned.

**Comment by ajitbohra** [comment]
Fixed in #51

**Comment by sistercylon** [comment]
I have tried installing this plugin to two different WP installations upgraded to 5.0 (public release; no other plugins installed). I get as far as clicking the Activate button and then receive the following message on a white screen:

Fatal error: Uncaught Error: Call to undefined function gutenberg_get_jed_locale_data() in /app/public/wp-content/plugins/gutenberg-examples-master/01-basic/index.php:57 Stack trace: #0 /app/public/wp-includes/class-wp-hook.php(286): gutenberg_examples_01_register_block('') #1 /app/public/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters(NULL, Array) #2 /app/public/wp-includes/plugin.php(453): WP_Hook->do_action(Array) #3 /app/public/wp-settings.php(467): do_action('init') #4 /app/public/wp-config.php(92): require_once('/app/public/wp-...') #5 /app/public/wp-load.php(37): require_once('/app/public/wp-...') #6 /app/public/wp-admin/admin.php(31): require_once('/app/public/wp-...') #7 /app/public/wp-admin/plugins.php(10): require_once('/app/public/wp-...') #8 {main} thrown in /app/public/wp-content/plugins/gutenberg-examples-master/01-basic/index.php on line 57

**Comment by karloscarweber** [comment]
I'm also finding this error. I guess it means that `wp_get_jed_locale_data` does not exist, and consequently it tries to call the `gutenberg_get_jed_locale_data` function, but it's not there. 

Interestingly the PR that fixes this, #51 , also has comments saying that It's not working in 5.0.1.

For whatever reason `wp_get_jed_locale_data` is completely missing from 5.0.1.

**Comment by SpencerCloud** [comment]
Just adding my voice that wp_get_jed_locale_data doesn't seem to exist in core anymore. Also tried searching the 5.0 files and it doesn't seem to be in there either.

**Comment by swissspidy** [comment]
That is correct. `wp_get_jed_locale_data` has been **removed**.

You should use `wp_set_script_translations()` instead.

**Comment by SpencerCloud** [comment]
It looks like `wp_get_jed_locale_data` has been replaced by a different function in core, `wp_set_script_translations` according to [this commit](https://github.com/WordPress/wordpress-develop/commit/0693a11592c1685c9000f04e06de84fc367cbae3).

I edited the files as a workaround so I could get to going through the tutorial files. For the first example, I edited 01-basic/index.php on line 47 where this code is in place:

`json_encode( ! function_exists( 'wp_get_jed_locale_data' ) ? gutenberg_get_jed_locale_data( 'gutenberg-examples' ) : wp_get_jed_locale_data( 'gutenberg-examples' ) )`

I changed the `wp_get_jed_locale_data` function to `wp_set_script_translations`, but since `wp_set_script_translations` requires at least 2 arguments ($handle and $domain), where the old function only needed 1, I added what seems to be the correct argument `'gutenberg-examples-01'` (the script handle) as the first argument, and kept the original argument `'gutenberg-examples'` (the text domain) as the second argument, so now the code on line 47 looks like this for me:

`json_encode( ! function_exists( 'wp_set_script_translations' ) ? gutenberg_get_jed_locale_data( 'gutenberg-examples' ) : wp_set_script_translations( 'gutenberg-examples-01' , 'gutenberg-examples' ) )`

It may or may not be the right fix for a patch, but it's working for me for now and the first example is showing up as a block under the layout category as it should.

I just did this for the first example, and I imagine a similar fix should work for the other examples, but I've commented out the other includes for now until I get to them, so now the index.php file in the top-level folder looks like this for me:

`include '01-basic/index.php';`
`// include '01-basic-esnext/index.php';`
`// include '02-stylesheets/index.php';`
`// include '03-editable/index.php';`
`// include '03-editable-esnext/index.php';`
`// include '04-controls/index.php';`
`// include '04-controls-esnext/index.php';`
`// include '05-recipe-card/index.php';`
`// include '05-recipe-card-esnext/index.php';`

It may not be the most elegant solution, but it's working for me and I'll do a similar temporary fix to just get through the examples.

If it is the right fix for a patch (minus commenting out the indcludes), I would be very proud.
